mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosFineGrainedSchedulerBackend.scala

```diff
@@ -59,6 +59,8 @@ private[spark] class MesosFineGrainedSchedulerBackend(
 
   private[mesos] val mesosExecutorCores = sc.conf.getDouble("spark.mesos.mesosExecutor.cores", 1)
 
+  private val useFetchCache = sc.conf.getBoolean("spark.mesos.fetchCache.enable", false)
```

Comment:
Please remove changes to the fine grained scheduler.  It's deprecated, and we shouldn't be adding any new features to it.